gadgetchains/phpThumb/FD/1/chain.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace GadgetChain\phpThumb;
+
+class FD1 extends \PHPGGC\GadgetChain\FileDelete
+{
+    public static $version = 'v1.7.12 <= v1.7.22';
```

Comment:
```suggestion
    public static $version = '<= v1.7.22';
```

Since you said it would work on all version before this one.

Reply:
Yup, we could do that.

Somewhat strangely It seems that v1.7.12 is the first release, at least in the project's current incarnation:

https://github.com/JamesHeinrich/phpThumb/tags